comp/process/expvars/expvarsimpl/expvars.go

```diff
@@ -59,6 +60,11 @@ func newExpvarServer(deps dependencies) (expvars.Component, error) {
 		return nil, err
 	}
 
+	if flavor.GetFlavor() == flavor.DefaultAgent {
```

Comment:
Should we have the condition check that if not process agent return nil? 

```go
if flavor.GetFlavor() != flavor.ProcessAgent {
    return nil, nil
}
```

It reads better the intention that we only want to initialize the server if is the process agent process

Reply:
makes sense to me. Done